Clinical-Genomics/scout
# fix missing demo bam files

This PR fixes #1256.

Reduced MT bam file is in demo data but it's not used in scout demo app. For instance when you look at variant: http://localhost:5000/cust000/643594/123c73baa49053a95b8f6058164a32e6
You see that bam file is missing:

![image](https://user-images.githubusercontent.com/28093618/60009945-13dc7380-9677-11e9-9c98-2790c130233b.png)

**How to test**:
1. git checkout to 'fix_demo_files_path' branch
1. run cli command **'scout setup demo'**
1. run **'scout --demo serve'** command and go to scout browser
1. go to variant http://localhost:5000/cust000/643594/123c73baa49053a95b8f6058164a32e6 and check that alignment buttons are visible (pileup and igv) and they point to variant viewer pages (you should be able to see the alignments).
1. BAM files for variants on other chromosomes than MT should not be found.

1. Try importing the test case by editing its config file (scout/demo/643595.config.yaml). Add a mt_bam and a bam_file for each of the 3 individuals in the file. Provide absolute path to the files.
1. Run:
```
scout --demo load case scout/demo/643595.config.yaml -u
```
and make sure that case is loaded correctly.
1.  run **'scout --demo serve'** command and go to scout browser
1. Make sure that variants and MT variants display buttons for pileup and IGV and that button links work.

**Expected outcome**:

- Alignment viewers buttons, when the bam files are available, should look like this:

![image](https://user-images.githubusercontent.com/28093618/60010158-7a619180-9677-11e9-9d34-1bae3ed5f9ec.png)

- Button links work and open alignment pages (both pileup and igv)


**Review:**
